src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "API",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1013"
+  },
+  "description": {
+    "text": "A project template for creating an ASP.NET Core application with an example Controller for a RESTful HTTP service. This template can also be used for ASP.NET Core MVC Views and Controllers.",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1014"
+  },
+  "order": 200,
+  "icon": "vs-2017.3/WebAPI.png",
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?LinkID=784882",
+  "uiFilters": [ "oneaspnet" ],
+  "supportsDocker": true,
+  "isApi": true,
+  "usesOidc": true,
+  "legacyTemplateIdentity": "Microsoft.NetCore.FSharp.WebAPI",
+  "supportedAuthentications": [
+    {
```

Comment:
I don't think we support auth for F# cc @javiercn

Reply:
It worked when I tried it out.  It doesn't do codegen, it just generates a json file with the specified authentication properties

Reply:
@jmarolf I'm not sure myself exactly what this affects, but having a JSON file with some auth settings doesn't enable auth... it might just be some random auth settings that nothing looks at...

Reply:
usesOidc doesn't actually do anything anymore, and legacyTemplateIdentity was primarily used to have consistent telemetry when the other ids changed. You can leave this one out.

supportedAuthentications controls what options are displayed in the One ASP.NET dialog. This is to provide some amount of decoupling between the template parameters and host specific parameters. If your template only supports NoAuth scenario, you can leave this out.